pmsv2/ewallet/params.go

```diff
@@ -0,0 +1,34 @@
+package ewallet
+
+type ChannelCode string
+
+type ChannelProperties map[string]interface{}
+
+const (
+	Grabpay   ChannelCode = "GRABPAY"
+	Paymaya   ChannelCode = "PAYMAYA"
+	Gcash     ChannelCode = "GCASH"
+	OVO       ChannelCode = "OVO"
+	DANA      ChannelCode = "DANA"
+	LinkAja   ChannelCode = "LINKAJA"
+	Shopeepay ChannelCode = "SHOPEEPAY"
+	Sakuku    ChannelCode = "SAKUKU"
+	Nexcash   ChannelCode = "NEXCASH"
+	Astrapay  ChannelCode = "ASTRAPAY"
+	Jeniuspay ChannelCode = "JENIUSPAY"
+)
+
+type CreateMethod struct {
```

Comment:
should only be channel_code and channel_properties also

Reply:
updated to be only channel_code and channel_properties